homeassistant/components/qbittorrent/switch.py

```diff
@@ -0,0 +1,112 @@
+"""Support for monitoring the qBittorrent API."""
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from qbittorrent import Client
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import QBittorrentDataCoordinator
+
+
+@dataclass(frozen=True, kw_only=True)
+class QBittorrentSwitchEntityDescription(SwitchEntityDescription):
+    """Describes QBittorren binary sensor entity."""
+
+    update_fn: Callable[["QBittorrentSwitch"], bool]
+    turn_on_fn: Callable[["QBittorrentSwitch"], None]
+    turn_off_fn: Callable[["QBittorrentSwitch"], None]
+
+
+def alternative_speed_turn_on(entity: "QBittorrentSwitch"):
+    """Turn on alternative speed.
+
+    The QBittorent only expose the toggle for alternative speed, so we test is alternative_speed is false before using toggle.
+    """
+
+    if not entity.is_on:
+        entity.client.toggle_alternative_speed()
+
+
+def alternative_speed_turn_off(entity: "QBittorrentSwitch"):
+    """Turn on alternative speed."""
+
+    if entity.is_on:
+        entity.client.toggle_alternative_speed()
+
+
+SWITCH_TYPES: tuple[QBittorrentSwitchEntityDescription, ...] = (
+    QBittorrentSwitchEntityDescription(
+        key="alternative_speed",
+        translation_key="alternative_speed",
+        icon="mdi:speedometer-slow",
+        update_fn=lambda entity: bool(entity.client.get_alternative_speed_status()),
+        # The QBittorent only expose the toggle for alternative speed, so we test is alternative_speed is false before using toggle
```

Comment:
```suggestion
        # qBittorrent only exposes the toggle for alternative speed, so we test if alternative_speed is false before using toggle.
```

Reply:
Was this meant to be here or was it meant to be an alternative version of the text in `alternative_speed_turn_off`?